mtags-interfaces/src/main/java/scala/meta/pc/reports/LazyReport.java

```diff
@@ -0,0 +1,5 @@
+package scala.meta.pc.reports;
+
+public interface LazyReport {
```

Comment:
Why do we need LazyReport? Could you add some comments about it?

Reply:
I realised I can use `Supplier` instead. I just wanted report creation to be lazy, since it won't always be used (verbose, EmptyReportContext). I'm not sure it's true now, but creating reports might require some calculations.